src/util/language.h

```diff
@@ -19,10 +19,14 @@ class symbol_tablet;
 class exprt;
 class namespacet;
 class typet;
+class cmdlinet;
 
 class languaget:public messaget
 {
 public:
+  // Parse language-specific options
+  virtual void get_language_options(const cmdlinet&) {}
```

Comment:
The functions that override this should be marked `override`.

https://travis-ci.org/diffblue/cbmc/jobs/196205293#L549